### PR TITLE
added-rotation-of-array-in-kotlin

### PR DESCRIPTION
**Pull request by sunny52525**
## Have you read the Contribution Guidelines?

>Yes

## Description

Left array rotation by specific number.

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
>Issue #494

### CODE REVIEW

**Comment by sunny52525** [comment]
@sukritishah15 kindly review the PR and merge it.

File: Kotlin/RotateArrayClockwise.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    for(i in 0..d-1){
+        var temp=arr[0];
+        for( j in 0..n-2){
+            arr[j]=arr[j+1];
+        }
+        arr[n-1]=temp;
+    }
+
+}
```
Comment (sukritishah15) [inline Kotlin/RotateArrayClockwise.kt:44]: @sunny52525 - Please add Sample I/O, Time and Space Complexity at the bottom in a multiline comment.

**Review by sukritishah15 — CHANGES_REQUESTED** [review]
Also, please add the file name and link in the README.md file present in the Kotlin folder as per the the format specified in the README.md file.

File: Kotlin/RotateArrayClockwise.kt (hunk 1 above)
Comment (sunny52525) [reply Kotlin/RotateArrayClockwise.kt:30]: changed as requested

**Comment by sunny52525** [comment]
> Also, please add the file name and link in the README.md file present in the Kotlin folder as per the the format specified in the README.md file.

done

**Review by sukritishah15 — APPROVED** [review]
Perfect!

**Pull request merged**